docs-2.0/nebula-operator/4.connect-to-nebula-graph-service.md

```diff
@@ -139,4 +144,93 @@ Steps:
+
+Ingress does not support TCP or UDP services. For this reason，the nginx-ingress-controller pod uses the flags `--tcp-services-configmap` and `--udp-services-configmap` to point to an existing ConfigMap where the key refers to the external port to be used and the value refers to the format of the service to be exposed. The format of the value `<namespace/service_name>:<service_port>`.
+
+For example, the configurations of the ConfigMap named as `tcp-services` is as follows:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: tcp-services
+  namespace: nginx-ingress
+data:
+  # update 
+  9769: "default/nebula-graphd-svc:9669"
+```
+
+Steps are as follows.
+
+1. Create a file named `nginx-ingress-daemonset-hostnetwork.yaml`. The YAML file reads as follows:
+
+  Click on [nginx-ingress-daemonset-hostnetwork.yaml](https://github.com/vesoft-inc/nebula-operator/blob/{{operator.branch}}/config/samples/nginx-ingress-daemonset-hostnetwork.yaml) to view the complete content of the example YAML file.
+
+  !!! note
+
+        The resource objects in the YAML file above use the namespace `nginx-ingress`. You can run `kubectl create namespace nginx-ingress` to create this namespace, or you can custom a different namespace.
+
+2. Label a node where runs the DaemonSet named `nginx-ingress-controller` in the above YAML file (The node used in this example is named `worker2` with an IP of `192.168.8.160`).
+
+  ```bash
+  kubectl label node worker2 nginx-ingress=true
+  ```
+
+3. Run the following command to enable Nginx Ingress in the cluster you created. 
+
+  ```bash
+  kubectl create -f nginx-ingress-daemonset-hostnetwork.yaml
+  ```
+
+  Output:
+
+  ```bash
+  configmap/nginx-ingress-controller created
+  configmap/tcp-services created
+  serviceaccount/nginx-ingress created
+  serviceaccount/nginx-ingress-backend created
+  clusterrole.rbac.authorization.k8s.io/nginx-ingress created
+  clusterrolebinding.rbac.authorization.k8s.io/nginx-ingress created
+  role.rbac.authorization.k8s.io/nginx-ingress created
+  rolebinding.rbac.authorization.k8s.io/nginx-ingress created
+  service/nginx-ingress-controller-metrics created
+  service/nginx-ingress-default-backend created
+  service/nginx-ingress-proxy-tcp created
+  daemonset.apps/nginx-ingress-controller created
+  ```
+
+  Since the network type is configured in Nginx Ingress is `hostNetwork`, after successfully deploying Nginx Ingress, you can access Nebula Graph through the IP (`192.168.8.160`) of the node where Nginx Ingress deployed and the external port (`9769`) you define.
+
+4. Run the following command to create a console used to connect to Nebula Graph and to access Nebula Graph databases with the IP (`192.168.8.160`) of the node where Nginx Ingress deployed and the external port you define.
```

Comment:
The sentence is a bit complex. Can we write it like this? 

Connect to Nebula Graph with Nebula Console. Use the IP address and port of Nebula Graph configured in the preceding steps.

Reply:
sure it is more concise